libraries/core_libs/consensus/src/dag/sortition_params_manager.cpp

```diff
@@ -2,44 +2,20 @@
 
 #include "pbft/pbft_block.hpp"
 
-namespace taraxa {
+#define NULL_BLOCK_HASH blk_hash_t(0)
```

Comment:
better would be to add this as a constant to constant.hpp

Reply:
Fixed